RapidReact/src/main/java/frc/robot/simulator/SimulatorTestSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.simulator;
+
+import frc.robot.Robot;
+import frc.robot.config.Config;
+import frc.robot.subsystem.BitBucketsSubsystem;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SimulatorTestSubsystem extends BitBucketsSubsystem {
+
+    final AtomicInteger counter = new AtomicInteger();
+
+    public SimulatorTestSubsystem(Config config) {
+        super(config);
+    }
+
+    @Override
+    public void init() {
+
+    }
+
+    @Override
+    public void periodic() {
+        int num = counter.incrementAndGet();
+
+        if (num > 700) {
+            System.exit(0);
+        }
+    }
```

Comment:
We should comment this method and the subsystem so other developers know why we have a subsystem that exists after 700 iterations.

Reply:
Comments are for losers